lms/djangoapps/course_api/blocks/transformers/milestones.py

```diff
@@ -0,0 +1,100 @@
+"""
+Milestones Transformer
+"""
+
+from django.conf import settings
+
+from edx_proctoring.api import get_attempt_status_summary
+from edx_proctoring.models import ProctoredExamStudentAttemptStatus
+from openedx.core.lib.block_structure.transformer import BlockStructureTransformer, FilteringTransformerMixin
+from util import milestones_helpers
+
+
+class MilestonesTransformer(FilteringTransformerMixin, BlockStructureTransformer):
+    """
+    Exclude proctored exams unless the user is not a verified student or has
```

Comment:
Yes, assuming the original code did. Lines 68-77 were copied directly from the old transformer:
`
# This section is an exam.  It should be excluded unless the
            # user is not a verified student or has declined taking the exam.
            user_exam_summary = get_attempt_status_summary(
                usage_info.user.id,
                unicode(block_key.course_key),
                unicode(block_key),
            )
            return user_exam_summary and (
                user_exam_summary.get('status', None) != ProctoredExamStudentAttemptStatus.declined
            )
`